submissions/KelechiPreciousNwachukwu/html-forms/contactform.html

```diff
@@ -14,7 +14,8 @@ <h1><a href = "#">PRECISE SPICE</a></h1>
             <nav>
                 <p>
                    <a href = "#">Account</a>
-                   <a href = "#">Spice Store</a>
+                   <a href = "../html\product-page.html">Spice Store</a>
+                   <a href = "..\html-forms\contactform.html">Contact Us</a>
```

Comment:
Check your slashes, is it correct?

Reply:
Hi @adaugochi it was actually navigating locally. I've effected the change, thanks.